quantum/quantum.h

```diff
@@ -110,7 +109,7 @@ extern uint32_t default_layer_state;
 #endif
 
 #ifdef TAP_DANCE_ENABLE
-  #include "process_tap_dance.h"
+    #include "process_tap_dance.h"
```

Comment:
```suggestion
  #include "process_tap_dance.h"
```
https://docs.qmk.fm/#/contributing?id=coding-conventions
2 spaces for indintation

Reply:
I'm fine with 4, and in fact we keep talking about changing that.

Reply:
I decided to stick with 4 spaces in this case since that's what most of the file already uses. I considered changing everything to 2 spaces but decided against it as it would be too many changes. Maybe in a future PR (or maybe not, given what @skullydazed said above).